src/System.Net.Security/src/System/Net/CertificateValidationPal.Unix.cs

```diff
@@ -227,5 +227,27 @@ private static X509Store EnsureStoreOpened(ref X509Store storeField, StoreLocati
 
             return store;
         }
+
+        private static int QueryContextRemoteCertificate(SafeDeleteContext securityContext, out SafeFreeCertContext remoteCertContext)
+        {
+            remoteCertContext = null;
+            try
+            {
+                SafeX509Handle remoteCertificate = Interop.OpenSsl.GetPeerCertificate(securityContext.SslContext);
+                // Note that cert ownership is transferred to SafeFreeCertContext
+                remoteCertContext = new SafeFreeCertContext(remoteCertificate);
+                return 0;
+            }
+            catch
+            {
+                return -1;
+            }
+        }
+
+        private static int QueryContextIssuerList(SafeDeleteContext securityContext, out Object issuerList)
```

Comment:
@rajansingh10 , please remove this in your next PR. I believe we don't use this anywhere